panel/models/datetime_slider.ts

```diff
@@ -0,0 +1,57 @@
+// adapted from bokeh
+// https://github.com/bokeh/bokeh/blob/branch-3.7/bokehjs/src/lib/models/widgets/sliders/date_slider.ts
+
+import tz from "timezone"
```

Comment:
External libraries like this one are not guaranteed to be importable by this name or a name altogether. I see that our date slider still uses the legacy approach. The right approach is to use `DEFAULT_FORMATTERS.datetime(value, format)` from `core/util/templating` instead. This way you also avoid adding type declarations for `timezone` library.

Reply:
Thanks @mattpap! That's good to know. Applied the changes as suggested.